install_sct.bat

```diff
@@ -52,15 +65,13 @@ echo:
 echo: 5. Click 'OK' three times. You can now access SCT's scripts in the Command Prompt.
 
 rem Return to initial directory and deactivate the virtual environment
-popd
-call deactivate
-
-exit /b 0
+goto exit
 
 :error
-set fatalerrorlevel=%errorlevel%

```

Comment:
Whoops, sorry! I had temporarily forgotten why this `fatalerrorlevel` pattern was necessary. (It preserves the error level so that subsequent commands don't overwrite this value.)

My bad! Let me fix this up quick.